vulture/core.py

```diff
@@ -255,6 +255,16 @@ def by_size(item):
         return sorted(confidently_unused,
                       key=by_size if sort_by_size else by_name)
 
+    def make_whitelist(self, min_confidence=0, sort_by_size=False):
+        for item in self.get_unused_code(
+                min_confidence=min_confidence, sort_by_size=sort_by_size):
+            if item.typ in ('function', 'class', 'variable'):
```

Comment:
What about properties and attributes? We need to add output for them as well.

Reply:
Can we handle it in a separate PR, please?

Reply:
I'd rather handle this in this PR since the functionality is tightly linked. I wouldn't want to see in the second PR that we have to undo some changes from the first PR.

What we need for properties and attributes (e.g., named `foo`) is the following output at the beginning of the output:

```
class VultureMock:
    def __getattr__(self, _):
        pass

_ = VultureMock()
```
and then for each unused property or attribute:
```
_.foo  # unused attribute (...)
```

Reply:
I was trying to access the parent node and extract it's name from there,  but this turns out to be way easier than it.😅

Thanks for the help! :-)